.evergreen/run-mongodb-aws-test.sh

```diff
@@ -60,8 +60,10 @@ authtest () {
     else
       . venvaws/bin/activate
     fi
+    sudo apt-get update || true
+    sudo apt-get install -y git || true
```

Comment:
We shouldn't run this because it will affect other tasks run on this host. Can we devise an alternative way to test against the "latest" version of pymongo-auth-aws vs the stable version? Perhaps we should switch all the current tests to use the latest version (using `pip install 'https://...'` so we don't need git) and then add a few tests for the stable version (installed with `pip install '.[aws]'`). We could also double the matrix. What do you think?

Reply:
I think we should test against released unless we are specifically testing a new feature, updated to use the zip download links.